kt-extensions/src/test/java/com/mindera/skeletoid/kt/extensions/utils/StringUnitTest.kt

```diff
@@ -232,4 +232,22 @@ class StringUnitTest {
 
         assertFalse(email.isEmailValid())
     }
+
+    @Test
+    fun testNullIfEmptyNull() {
+        val text: String? = null
+        assertEquals(null, text.nullIfEmpty())
+    }
+
+    @Test
+    fun testNullIfEmptyEmpty() {
+        val text: String = ""
+        assertEquals(null, text.nullIfEmpty())
+    }
+
+    @Test
+    fun testNullIfEmpty() {
+        val text: String = "text"
+        assertEquals("text", text.nullIfEmpty())
+    }
```

Comment:
Are the tests for nullIfBlank somewhere else? If so maybe we can move them here? If there aren't any maybe we can add them here (since they are very similar to these)

Reply:
(I'll approve it since this isn't actually a problem with this PR and you can just create a task and resolve the conversation if you don't want to do it now)